### PR TITLE
styling-Tanisha

### PR DESCRIPTION
**Pull request by Tanisha07**
For #50

### CODE REVIEW

File: participants/Tanisha07.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-Name: Tanisha Sharda
-GitHub username: Tanisha07
-University: IGDTUW
-Year: 2
-Branch: IT
+______________________________________
+ #### `**Name:** Tanisha Sharda`
```
Comment (lisa761) [inline participants/Tanisha07.md:2]: Hi @Tanisha07, the bold using ** does not work inside ``.

Comment (Tanisha07) [reply participants/Tanisha07.md:2]: Okay, yes. I realized this after viewing the file on StackEdit and made changes there but forgot to update those changes and on my file. I'll update it again.

**Comment by lisa761** [comment]
Hi @Tanisha07, Do you think perhaps, that your PR should be named differently? Anyone can mistake it for a submission of Task-2,  but I rechecked and you have already given a successful submission for it. Please take care of this next time.

**Comment by Tanisha07** [comment]
Hi @lisa761. Thanks for informing. I'll be more careful in future.

**Pull request merged**